weasel/cli/dvc.py

```diff
@@ -1,4 +1,4 @@
-"""This module contains helpers and subcommands for integrating spaCy projects
+"""This module contains helpers and subcommands for integrating Weasel
 with Data Version Controk (DVC). https://dvc.org"""
```

Comment:
```suggestion
with Data Version Control (DVC). https://dvc.org"""
```
This typo just caught my eye...